client/src/services/participant.js

```diff
@@ -1,8 +1,8 @@
 import store from 'store';
 import { API_URL, pageSize, postHireStatuses } from '../constants';
 
-const getCohortName = (cohort = {}) =>
-  cohort.cohort_name && cohort.psi?.institute_name
+export const getCohortPsiName = (cohort = {}) =>
+  cohort && cohort.cohort_name && cohort.psi?.institute_name
```

Comment:
`cohort?.cohort_name &&`
?

Reply:
Yep, that's probably better ☝️
I'll update it